components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -354,7 +354,6 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
         for (const team of teams) {
             allProjects.push(...(await this.projectsService.getTeamProjects(team.id)).map((p) => p.id));
         }
-        allProjects.push(...(await this.projectsService.getUserProjects(this.userID)).map((p) => p.id));
```

Comment:
Uh, missed this line :see_no_evil: 
@svenefftinge We are not relying on this somewhere?

Reply:
Ok, we _should_ not :crossed_fingers: